common/persistence/src/test/java/app/coronawarn/server/common/persistence/utils/hash/HashUtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package app.coronawarn.server.common.persistence.utils.hash;
+
+import app.coronawarn.server.common.persistence.utils.hash.HashUtils.MessageDigestAlgorithms;
+import com.google.protobuf.ByteString;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class HashUtilsTest {
```

Comment:
The test is missing some scenarios for "byteStringDigest". Maybe we can add an additional test to cover that?

Reply:
I will add these tests in the following PR regarding shared common module as both Utils and Test class will be re-located.